src/utils/create-review-card.js

```diff
@@ -0,0 +1,17 @@
+const createReviewCard = (review) => {
+    return (
+    
+        <section  key={review.review_id} className='item'>
+                    <h4 className='review-card-title'>{review.title}</h4>
+                    <p className='review-card-author'>{review.owner}</p>
+                    <p className='review-card-blurb'>{review.review_body.slice(0,200)}...</p>
+                    <p className='review-card-category'>{review.category}</p>
+                    <img src={review.review_img_url} alt={review.title} className='review-card-img' ></img>
+                    <p className='review-card-votes'> Votes: {review.votes}</p>
+                    <p className='review-card-comments'>Commment Count: {review.comment_count}</p>
+                    <p className='review-card-date'>Added {review.created_at}</p>
+                </section>
+    )
+}
+
+export default createReviewCard
```

Comment:
This isn't the way we should be making react components

You should name a component after what UI it produces - instead of `createReviewCard` it should be `ReviewCard`

You should be rendering it like an element - `<ReviewCard />` - not invoking it like you are in `reviews-by-caregory.jsx` 

``diff
{currReviews.map((review) => {
-    return createReviewCard(review);
+    return <ReviewCard review={review}/>
})}
```